.travis.yml

```diff
@@ -9,5 +9,10 @@ cache:
   yarn: true
   directories:
     - node_modules
-script:
-  - bin/test
+
+matrix:
```

Comment:
So is this doubling our build times because we're building it twice? Is there a way we can run the test without building jekyll twice?

Reply:
We can do it in one build but then we have to run tests and linkcheck sequentially. I am not sure what's faster.

link check is definetly much slower than any build procedure though

Reply:
Ah, I mean I guess we can do one build and parallelize within the same travis job, but then the output is mixed

Reply:
I guess we can see how it works out.